pkg/lifecycle/startup/startup_test.go

```diff
@@ -52,6 +53,27 @@ func fakeExitFunction(ec int) {
 	exitCode = ec
 }
 
+func getDefaultInterfacesToExclude() []string {
```

Comment:
I think this function can be removed if we revert the change to pass the interfaces as an argument.

Reply:
yes removed it.